framework/test/test_app/app.js

```diff
@@ -21,7 +21,7 @@ const {
 	/* eslint-disable import/no-unresolved */
 } = require('../../src');
 
-const packageJSON = require('../../package');
+const packageJSON = require('../../../lisk/package.json');
```

Comment:
lisk package will be removed, so it shouldn't reference it

Reply:
Shall I hard-code these values here?

Reply:
mmm yes let's hard-code this for now...